### PR TITLE
Implement image visualization tab in run detail page

### PR DESCRIPTION
**Pull request by VkoHov**
- Add images tab in run detail page
- Implement images visualization

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#948](https://github.com/aimhubio/aim/issues/948)

File: aim/web/ui/src/services/models/runs/util.ts [hunk 1]
```diff
@@ -218,8 +226,66 @@ export function processDistributionsData(data: Partial<DistributionsData>) {
...
-export function processImagesData() {
-  return {};
+export function processImagesData(data: Partial<ImagesData>, params?: object) {
+  const { record_range, iters, values, index_range, context, name } = data;
+  const groupingSelectOptions = params
+    ? imagesExploreAppModel.getGroupingSelectOptions({
+        params: getObjectPaths(params as any, params as any),
+      })
+    : [];
+  let metrics: any[] = [];
```
Comment (roubkar) [inline aim/web/ui/src/services/models/runs/util.ts:236]: why `metrics` ?
also let's provide types

**Review by roubkar — APPROVED** [review]

**Review by rubenaprikyan — APPROVED** [review]
Great job!

File: aim/web/ui/src/services/models/runs/util.ts (hunk 1 above)
Comment (VkoHov) [reply aim/web/ui/src/services/models/runs/util.ts]: Done

**Pull request merged**